posthog/models/web_preaggregated/sql.py

```diff
@@ -90,6 +100,14 @@ def DISTRIBUTED_WEB_STATS_DAILY_SQL():
     return DISTRIBUTED_TABLE_TEMPLATE("web_stats_daily_distributed", "web_stats_daily", WEB_STATS_COLUMNS)
 
 
+def WEB_BOUNCES_DAILY_SQL(table_name="web_bounces_daily", on_cluster=True):
```

Comment:
Do we use a different `table_name` somewhere?

Reply:
Nope, it was just a helper to use the same base template with at least (team_id, device_type, host, day_bucket) that ended up on the INSERT_SQL as well, made a note to refactor. I think I will need three more tables (curren_urls, exit_paths and outbound clicks), so I will write a single PR to refactor it.